setup.py

```diff
@@ -36,7 +35,7 @@
     packages=find_packages(exclude=('tests',)),
     include_package_data=True,
     install_requires=[
-        'django-pymemcache>=1.0',
-        'Django>=2.2',
+        'pymemcache==3.3.0',
```

Comment:
I'd prefer a newer version like 3.5.0 but pinterest/pymemcache#353 causes breakage so I stick with a version that's been used before